code/modules/mob/living/carbon/life.dm

```diff
@@ -270,14 +270,16 @@
 
 	if(..()) //alive
 
-		if(health <= config.health_threshold_dead || !getorgan(/obj/item/organ/brain))
+		if(!getorgan(/obj/item/organ/brain))
 			death()
 			return
-
-		if(getOxyLoss() > 50 || health <= config.health_threshold_crit)
-			Paralyse(3)
-			stat = UNCONSCIOUS
-
+		if(getBrainLoss() >= 100 && stat != DEAD)
+			Weaken(20)
+			losebreath += 10
+			silent += 2
+		if(getBrainLoss() >= 120 && stat != DEAD)

```

Comment:
Will have to make adjustments to the cloner if you go with this, otherwise cloned people will immediately drop dead from starting off with 190 brain damage.